catch_2.py

```diff
@@ -57,6 +57,12 @@ def refresh_items(file="items.json"):
 while unfitted_items != 0:
     packing_efficacy = 0
 
+    for b in packer.bins:
+       b.items.clear()
```

Comment:
Maybe better create function in Packer class to clear its stuff. If later we would need to clear more stuff then these two lists, we will add simply code there.

Reply:
Tru ;-)